components/default-settings/src/main/resources/XWiki/XWikiPreferences.xml

```diff
@@ -1413,9 +1413,21 @@
         <unmodifiable>0</unmodifiable>
         <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
       </AllowPushesFromNonListedServers>
+      <AllowNoConsentsFromOldClients>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <name>AllowNoConsentsFromOldClients</name>
+        <number>1</number>
+        <prettyName>AllowNoConsentsFromOldClients</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </AllowNoConsentsFromOldClients>
     </class>
     <property>
-      <AllowPushesFromNonListedServers>1</AllowPushesFromNonListedServers>
+      <AllowPushesFromNonListedServers>0</AllowPushesFromNonListedServers>
```

Comment:
If we include push functionality by default (i.e. it will always be listening for pushes), we don't want people to receive pushes they don't expect - of course, one needs a username and password, but even then maybe they don't want that. So this is a way to disable push while still including the component